src/API/REST/V3/Routes/Donations/DonationController.php

```diff
@@ -639,7 +640,7 @@ public function permissionsCheck(WP_REST_Request $request)
      */
     public function update_item_permissions_check($request)
     {
-        if (!current_user_can('manage_options')) {
+        if (!current_user_can('edit_give_payments') && !current_user_can('manage_options')) {
```

Comment:
Shouldn't this be an _or_ check?  

1. If admin then pass
2. if can't edit payment then fail

```suggestion
        if (!current_user_can('edit_give_payments') || !current_user_can('manage_options')) {
```